### DISCUSSION TITLE
Potential feature: Call `dev`/`start` manually in code

### DISCUSSION THREAD

**Discussion by RATIU5** [nksaraf/vinxi#444] [Ideas]
I'm working on a project where this would be a lifesaver. Instead of calling `npm run vinxi dev` or `npm run vinxi start`, having the ability to run the code those scripts run manually in my code would help me so much. I don't want to rely on using the `vinxi` CLI script as I already have my own CLI that needs to run other commands/code. Has anyone else ran into a problem where this could help? I want to see if it's worth it to look into this feature request.

**Comment by Mirza-Glitch** [comment]
I just found out about [App Api](https://vinxi.vercel.app/api/app.html), i haven't tried this but it might be a lifesaver
